src/modules/Builder/consts.ts

```diff
@@ -0,0 +1,2 @@
+//TODO: remove in all places where it was used after the next major react-hook-form update
```

Comment:
maybe add more details for exact version:
```suggestion
//TODO: remove in all places where it was used after release the v.8.0.0 react-hook-form
```

Reply:
done